charts/ingress-nginx/values.yaml

```diff
@@ -93,6 +93,8 @@ controller:
   # when users add those annotations.
   # Global snippets in ConfigMap are still respected
   allowSnippetAnnotations: false
+  # -- This configuration defines if NGINX workers should reload serially instead of concurrently when multiple changes that require reloads are queued
+  enableSerialReloads: false
```

Comment:
Let's change this to `enableWorkerSerialReloads` to make it more clearly. Thanks

Reply:
done :)